SpatialConnect/SCBackendService.h

```diff
@@ -39,7 +42,8 @@
 @property(readonly, strong) RACBehaviorSubject *configReceived;
 
 - (id)initWithRemoteConfig:(SCRemoteConfig *)cfg;
-
+- (void)registerForLocalNotifications;
+- (void)createNotification:(SCNotification *)notification;
```

Comment:
These should be private if they are only used in the .m file. You can add them to the interface in the .m file.